.github/workflows/ci.yaml

```diff
@@ -1013,3 +1013,23 @@ jobs:
     uses: ./.github/workflows/build-msi.yml
     if  : github.event_name == 'pull_request' && contains(github.event.pull_request.body, '[test_msi]')
     # TODO: ADD A JOB THAT DEPENDS ON THIS TO TEST THE MSI
+    
+  build-sdk-package:
+    uses: ./.github/workflows/build-sdk.yml
+    with:
+      java-version: 8
+
+  build-chocolatey-package:
+    uses: ./.github/workflows/build-chocolatey.yml
+    needs: [ build-sdk-package ]
+    with:
+      version: 3.6.0-local # TODO: FIX THIS
```

Comment:
TODO?

Reply:
Yes, we should provide a version number for the package we are locally building, it is not important because we never publish that package but I want to find a better pattern for this. I tried `local` only but it fails because of the spec. Should not be considered a blocker.